### DISCUSSION TITLE
Allow —force for npm install

### DISCUSSION THREAD

**Discussion by azro352** [renovatebot/renovate#27231] [Suggest an Idea]
Hi
Sometimes we're required to have unmaintained packages in our project, ones that have specified versions of react that aren't the latests
That implies to need the `--force` option at every npm install step, but we know the risk

It would be nice if renovate could do it too, because as it doesn't now, it fails to update the package lock and no MR is opened 

Do you agree ? 

If you do, I'll open a PR to add a `postUpdateOptions` values at https://github.com/renovatebot/renovate/blob/main/lib/config/options/index.ts#L2251
And the corresponding use at https://github.com/renovatebot/renovate/blob/main/lib/modules/manager/npm/post-update/npm.ts#L102

Thanks

**Comment by rarkins** [comment]
Can it be set in npmrc? Can it be set using env variables?

**Reply by azro352** [reply]
Not in env, but seems yes in `npmrc` but I already have a `npmrc` section in my global config, if i set it with `force=true` in the projetc, it will ovewrite the global `npmrc` item

**Reply by rarkins** [reply]
You can configure npmrc at repo level too. If you wish to merge them with a global npmrc then set npmrcMerge=true.

Then it is not necessary/suitable as a postUpdateOption
